src/components/SidePanel/SidePanel.tsx

```diff
@@ -33,6 +33,16 @@ export type Props = {
    */
   hasError?: boolean;
 
+  /**
+   * Whether or not the side panel animates on open or close. This requires the side panel to be controlled by the `isOpen` prop instead of conditional rendering.
```

Comment:
```suggestion
   * Whether the side panel animates on open or close. This requires the side panel to be controlled by the `isOpen` prop instead of conditional rendering.
```
A bit more concise :)